pkg/container/qemu_runner.go

```diff
@@ -370,6 +373,19 @@ func createMicroVM(ctx context.Context, cfg *Config) error {
 		baseargs = append(baseargs, "-cpu", cfg.CPUModel)
 	} else {
 		baseargs = append(baseargs, "-cpu", "host")
+		// we cant use `-cpu host` when architecture does not match between host
+		// and guest
+		if cfg.Arch.ToAPK() != apko_types.ParseArchitecture(runtime.GOARCH).ToAPK() {
+			// let's use a recent default for those
+			baseargs = append(baseargs, "-cpu", "max")
```

Comment:
I think this will end up being confusing, since the paths below max it 100% unused

Reply:
```suggestion
```